ts/components/messages/MessagesArchive.tsx

```diff
@@ -168,13 +189,37 @@ class MessagesArchive extends React.PureComponent<Props, State> {
               bordered={true}
               light={true}
               onPress={resetSelection}
-              style={styles.buttonBarSecondaryButton}
+              style={
+                isExperimentalFeaturesEnabled
```

Comment:
We don't need to use experimentalFeature flag for this feature.

Reply:
@fpersico done in commit 16bf7dc